src/pow.cpp

```diff
@@ -35,12 +35,9 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
             return 0x1e0ffff0;
         } else if(nHeight >= 1080000 && nHeight < 1080010) { // Force difficulty for 10 blocks - Lyra2REv3 hardfork
             return 0x1b0ffff0;
-        } 
-        /* 
-            else if(nHeight >= VERTHASH_FORKBLOCK_MAINNET && nHeight < VERTHASH_FORKBLOCK_MAINNET+10) { // Force difficulty for 10 blocks - Verthash hardfork
-                return 0x1b0ffff0;
-            }
-        */
+        } else if(nHeight >= VERTHASH_FORKBLOCK_MAINNET && nHeight < VERTHASH_FORKBLOCK_MAINNET+10) { // Force difficulty for 10 blocks - Verthash hardfork
+            return 0x1c07fff8;
```

Comment:
How is this number calculated?

Reply:
See the opening comments of the PR for the factor 128 explanation.

I calculated the factor 128 against the previous hardfork difficulty like this:

* Convert compact bits to bignum target
* Multiply resulting bignum by 128
* Convert back to compact bits

https://play.golang.org/p/TVmR62Gr071